eventstore/dynamo/dynamo.go

```diff
@@ -0,0 +1,193 @@
+package dynamo
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/hallgren/eventsourcing/core"
+)
+
+type DynamoEventStore struct {
+	db        *dynamodb.DynamoDB
+	tableName string
+}
+
+func New(sess *session.Session, tableName string) *DynamoEventStore {
+	return &DynamoEventStore{
+		db:        dynamodb.New(sess),
+		tableName: tableName,
+	}
+}
+
+func (store *DynamoEventStore) Save(events []core.Event) error {
+	// Comprobar si hay eventos
+	if len(events) == 0 {
+		return nil
+	}
+
+	lastVersion, err := store.getLastVersion(events[0].AggregateID)
+	if err != nil {
+		// Manejar error al obtener la última versión
+		return err
+	}
+
+	// Comprobar si la versión del evento es la siguiente versión esperada
+	if core.Version(lastVersion)+1 != events[0].Version {
+		return core.ErrConcurrency
+	}
+
+	for i := range events {
```

Comment:
This is probably a matter of taste, but I think it's preferable to iterate like this instead: 
```suggestion
	for _, event := range events {
```

And instead refer to the event as simply `event` instead of `event[i]`,.